sirepo/package_data/static/js/radia.js

```diff
@@ -1808,9 +1808,23 @@ SIREPO.app.directive('fieldLineoutReport', function(appState) {
             };
 
             $scope.$on('radiaViewer.loaded', () => {
+                if ($scope.dataCleared) {
+                    $scope.simState.runSimulation();
+                }
                 $scope.dataCleared = false;
             });
 
+            $scope.$on('fieldLineoutAnimation.changed', function () {
+                if (! $scope.dataCleared && $scope.hasPaths()) {
```

Comment:
Make `! $scope.dataCleared && $scope.hasPaths()` into a named function so it is clear what it is testing for and use it here and in the template above. Also, all of these ifs can be collapsed into one.